### PR TITLE
Better wording for the error reported by the rule "no-else-return" #6411

### PR DESCRIPTION
**Pull request by othomann**
This is the requested pull request for the opened issue #6411. I also took the time to remove the warning in the file related to the rule "no-else-return". I hope this is ok.

### CODE REVIEW

**Comment by othomann** [comment]
I think this is ready now.

File: lib/rules/no-else-return.js [hunk 1]
```diff
@@ -112,14 +112,13 @@ module.exports = {
 
                 // If we have a BlockStatement, check each consequent body node.
                 return node.body.some(checkForReturnOrIf);
-            } else {
-
-                /*
-                 * If not a block statement, make sure the consequent isn't a
-                 * ReturnStatement or an IfStatement with returns on both paths.
-                 */
-                return checkForReturnOrIf(node);
```
Comment (vitorbal) [inline lib/rules/no-else-return.js]: good catch that we don't need the `else`!

**Comment by vitorbal** [comment]
Should be `Update:` instead of `Fix:` since it's an enhancement, not a bug. Also the commit message must be 72 characters in length or shorter.

Otherwise, LGTM.

**Comment by vitorbal** [comment]
Adding "do not merge" for now just until #6411 gets accepted.

**Comment by othomann** [comment]
So I must update the commit message to get it delivered ?

**Comment by ilyavolodin** [comment]
@othomann No, we just need to mark the issue as accepted at this point.

**Comment by vitorbal** [comment]
thanks for the contribution @othomann! 👍

**Pull request merged**